vic/drivers/cesm/cpl_esmf/lnd_comp_mct.F90

```diff
@@ -0,0 +1,224 @@
+module lnd_comp_mct
+
+! !USES:
+  use shr_kind_mod, only: IN=>shr_kind_IN
+  use seq_infodata_mod
+  use seq_timemgr_mod
+  use mct_mod
+  use ESMF
+
+  use seq_cdata_mod
+  use seq_comm_mct     , only: seq_comm_getcompstates
+
+  use esmfshr_mod
+  use lnd_comp_esmf
+!
+! !PUBLIC TYPES:
+  implicit none
+  save
+  private ! except
+
+!--------------------------------------------------------------------------
+! Public interfaces
+!--------------------------------------------------------------------------
+
+  public :: lnd_init_mct
+  public :: lnd_run_mct
+  public :: lnd_final_mct
+  public :: lnd_register
+
+!--------------------------------------------------------------------------
+! Private data interfaces
+!--------------------------------------------------------------------------
+
+!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+CONTAINS
+!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+subroutine lnd_register(lnd_petlist, ccsmComp, lnd_comp, import_state, export_state)
+
+   implicit none
+
+   ! Arguments
+   integer, pointer                  :: lnd_petlist(:)
+   type(ESMF_CplComp)                :: ccsmComp
+   type(ESMF_GridComp),intent(out)   :: lnd_comp
+   type(ESMF_State)   ,intent(out)   :: import_state, export_state
+
+  ! Local variables
+   integer            :: rc
+
+  ! Create Gridded Component and import/export States
+   lnd_comp = ESMF_GridCompCreate(name="lnd_comp", petList=lnd_petlist, rc=rc)
```

Comment:
In `vic/drivers/cesm/cpl_esmf/lnd_comp_esmf.F90` you use `if (` rather than `if(`. Pick a style and use it consistently